mocks/mock_bango_payment_account.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
```

Comment:
Nit: Remove the shebang, it's not needed for files that are not executed directly.